packages/nextcloud/test/core_test.dart

```diff
@@ -1,14 +1,16 @@
+@Retry(3)
+library core_test;
```

Comment:
Why this?

can't we add the annotation to the main method?

Reply:
The annotation can only be set on a library directive. I also was confused by this at first.